Gopkg.toml

```diff
@@ -74,25 +74,33 @@ required = [
   name = "github.com/prometheus/client_golang"
   version = "0.9.3"
 
-[[constraint]]
+[[override]]
+  version = "v1.5.2"
+  name = "github.com/russross/blackfriday"
+
+[[override]]
   name = "k8s.io/api"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.0"
 
-[[constraint]]
+# [[override]]
+#   name = "k8s.io/apiextensions-apiserver"
+#   version = "kubernetes-1.15.0"
```

Comment:
Build fails with the error below. I tried to fix it by setting the dependency above but then dep can't resolve the dependencies due to a problem with kustomize. I'm blocked here and need some help.

```
# github.com/giantswarm/helmclient/vendor/k8s.io/apiextensions-apiserver/pkg/features
vendor/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go:35:27: undefined: feature.Feature
vendor/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go:42:29: undefined: feature.Feature
vendor/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go:48:34: undefined: feature.Feature
vendor/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go:52:32: feature.DefaultFeatureGate.Add undefined (type featuregate.FeatureGate has no field or method Add)
vendor/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go:58:41: undefined: feature.Feature
vendor/k8s.io/apiextensions-apiserver/pkg/features/kube_features.go:58:61: undefined: feature.FeatureSpec
```